### PR TITLE
LOOP-1649: Therapy Settings copy changes

### PR DESCRIPTION
**Pull request by rickpasetto**
Copy changes from https://tidepool.atlassian.net/browse/LOOP-1649 and https://tidepool.atlassian.net/browse/IFU-24

Also adds guardrail "information text" to GlucoseTherapySettingInformationView
Also adds a common framework for displaying both mg/dL and mmol/L in a text string

### CODE REVIEW

File: Extensions/Guardrail+Settings.swift [hunk 1]
```diff
@@ -34,29 +34,30 @@ public extension Guardrail where Value == HKQuantity {
...
             suspendThreshold?.quantity
         ]
         .compactMap { $0 }
         .max()!
         let recommmendedLowerBound = max(absoluteLowerBound, correctionRangeScheduleRange.upperBound)
         return Guardrail(
-            absoluteBounds: absoluteLowerBound...workoutCorrectionRange.absoluteBounds.upperBound,
-            recommendedBounds: recommmendedLowerBound...workoutCorrectionRange.recommendedBounds.upperBound
+            absoluteBounds: absoluteLowerBound...unconstrainedWorkoutCorrectionRange.absoluteBounds.upperBound,
+            recommendedBounds: recommmendedLowerBound...unconstrainedWorkoutCorrectionRange.recommendedBounds.upperBound
```
Comment (nhamming) [inline Extensions/Guardrail+Settings.swift:53]: are there tests for these changes?

File: LoopKit/DeliveryLimits.swift [hunk 2]
```diff
@@ -44,12 +44,12 @@ public extension DeliveryLimits.Setting {
         }
     }
     
-    var descriptiveText: String {
+    func descriptiveText(appName: String) -> String {
```
Comment (nhamming) [inline LoopKit/DeliveryLimits.swift]: Is it common that descriptiveText is assumed to be localized? If not, consider adding `localized` to the func name to make that clear.

File: LoopKit/TherapySetting.swift [hunk 3]
```diff
@@ -58,23 +58,23 @@ public extension TherapySetting {
...
         case .suspendThreshold:
             return String(format: LocalizedString("%1$@ will deliver basal and recommend bolus insulin only if your glucose is predicted to be above this limit for the next three hours.", comment: "Descriptive format string for glucose safety limit (1: app name)"), appName)
         case .basalRate:
-            return LocalizedString("Your basal rate of insulin is the number of units per hour that you want to use to cover your background insulin needs.", comment: "Descriptive text for basal rate")
+            return LocalizedString("Your Basal Rate of insulin is the number of units per hour that you want to use to cover your background insulin needs.", comment: "Descriptive text for basal rate")
         case .deliveryLimits:
-            return "\(DeliveryLimits.Setting.maximumBasalRate.descriptiveText)\n\n\(DeliveryLimits.Setting.maximumBolus.descriptiveText)"
+            return "\(DeliveryLimits.Setting.maximumBasalRate.descriptiveText(appName: appName))\n\n\(DeliveryLimits.Setting.maximumBolus.descriptiveText(appName: appName))"
```
Comment (nhamming) [inline LoopKit/TherapySetting.swift]: Here is the root of my other comment. It is clear that localized string is expected, but not clear that it is being provided without jumping to that code.

**Review by nhamming — APPROVED** [review]
Just a general comment about adding `localized` to functions that return localized strings. This is a pattern that already exists and my preference is that this is kept consistent. But I don't hold this position strongly. Otherwise LGTM.

File: Extensions/Guardrail+Settings.swift (hunk 1 above)
Comment (rickpasetto) [reply Extensions/Guardrail+Settings.swift:53]: Strictly speaking...yes.  This file's only change was to "expose" already-established local vars as (internal) vars. I should have annotated the PR with that...

**Pull request merged**